src/store/getters/state-getters.ts

```diff
@@ -254,8 +254,9 @@ export const isFilingValid = (state: any): boolean => {
     state.stateModel.staffPaymentStep.valid)
 }
 
-export const getOriginalIAFilingDate = (state: any): boolean => {
-  return state.stateModel.originalIA.header.date.split('T')[0]
+export const getDefaultCorrectionDetailComment = (state: any): string => {
+  const filingDate = state.stateModel.originalIA.header.date.split('T')[0]
```

Comment:
Should you have a fallback value if original IA has not been loaded yet? (I know the object exists in the state model but there date=''.)

Reply:
Great call!